test_scripts/Defects/5_0/1006/1006_SDL_responses_with_GENERIC_ERROR_instead_of_UNSUPPORTED_RESOURCE_check_for_the_AddCommand.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1006
+-- Description: PoliciesManager must allow all requested params in case "parameters" field is omitted
+-- Precondition:
+-- 1) SDL and HMI are started.
+-- In case:
+-- 1) Send any single UI-related RPC , UI interface is not supported by the system
+-- 2) SDL receives UI.IsReady (available=false) from HMI
+-- Expected result:
+-- 1) SDL must respond "UNSUPPORTED_RESOURCE, success=false, info: UI is not supported by system" to mobile app
+-- Actual result:
```

Comment:
Please remove the actual result

Reply:
@GetmanetsIrina You can find the updates in the commit ac51975